content/deploy/centos.yaml

```diff
@@ -535,7 +539,7 @@ chapters:
                 node7-cb                  not created (virtualbox)
                 ```
 
-              - Volià! We've got an extra Couchbase Server node. Let's bring it up and add it to the cluster. This time, let's use the REST API to add it into the cluster.
+              - Voilà! We've got an extra Couchbase Server node. Let's bring it up and add it to the cluster. This time, let's use the REST API to add it into the cluster.
```

Comment:
Well this one is just embarrassing! I even looked up the accent!

Reply:
Oh la la. ;)